sdk/core/Microsoft.Azure.Core.NewtonsoftJson/src/Microsoft.Azure.Core.NewtonsoftJson.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Core" />
+    <ProjectReference Include="../../Azure.Core/src/Azure.Core.csproj" />
```

Comment:
Is the idea to undo this once `Azure.Core` with these changes ships? should a TODO be added to remember?

Reply:
Added